cmd/package_test.go

```diff
@@ -363,27 +573,65 @@ author: package-author
 }
 
 func (s *suite) importFakeDemoPkg(c *C) {
-	packageYamlText := `
-name: fake.demo
-title: Fake Demo
-author: Demo Author
-`
-	runYamlText := `
-runtime: native
-config_set:
-  demoBoot1:
-    bootcmd: echo Demo1
-  demoBoot2:
-    bootcmd: echo Demo2
-`
+	packageYamlText := fixIndent(`
+		name: fake.demo
+		title: Fake Demo
+		author: Demo Author
+	`)
+	runYamlText := fixIndent(`
+		runtime: native
+		config_set:
+		  demoBoot1:
+		    bootcmd: echo Demo1
+		  demoBoot2:
+		    bootcmd: echo Demo2
+	`)
 	files := map[string]string{
 		"/meta/package.yaml":            packageYamlText,
 		"/meta/run.yaml":                runYamlText,
 		"/meta/README.md":               DefaultText,
 		"/fake-demo-file.txt":           DefaultText,
 		"/data/fake-demo-data-file.txt": DefaultText,
 	}
+	s.importPkg(files, c)
+}
+
+func (s *suite) importFakeDemoPkgWithRunYaml(runYamlText string, c *C) {
+	packageYamlText := fixIndent(`
+		name: fake.demo
+		title: Fake Demo
+		author: Demo Author
+	`)
+
+	files := map[string]string{
+		"/meta/package.yaml": packageYamlText,
+		"/meta/run.yaml":     runYamlText,
+	}
+	s.importPkg(files, c)
+}
+
+func (s *suite) importPkg(files map[string]string, c *C) {
 	tmpDir := c.MkDir()
 	PrepareFiles(tmpDir, files)
 	ImportPackage(s.repo, tmpDir)
 }
+
+func (s *suite) requireFakeDemoPkg(c *C) {
+	packageYamlText := fixIndent(`
+		name: package-name
+		title: PackageTitle
+		author: package-author
+		require:
+		  - fake.demo
+	`)
+	ioutil.WriteFile(filepath.Join(s.packageDir, "meta", "package.yaml"), []byte(packageYamlText), 0700)
+}
+
+func (s *suite) setRunYaml(runYamlText string, c *C) {
+	ioutil.WriteFile(filepath.Join(s.packageDir, "meta", "run.yaml"), []byte(runYamlText), 0700)
+}
+
+func fixIndent(s string) string {
+	r := regexp.MustCompile("\t")
```

Comment:
Is using regex more efficient than just using `strings.replace`?

This can then become oneliner

```
return strings.Replace(s, "\t", "", -1)
```

https://play.golang.org/p/QokAcNENa2